exc1/src/main/java/dgodek/company/TeamManager.java

```diff
@@ -0,0 +1,91 @@
+package dgodek.company;
+
+/**
+ * Created by matematyk60 on 23.07.17.
+ */
+public class TeamManager extends AbstactEmployee implements Manager {
+    private Employee[] employees;
+
+    private int number_of_employees;
+
+    private int max_size;
```

Comment:
I think since the requirements mentioned an array, its size should be unmodifiable - that is, the max size of employee a manager can manage is fixed at creation time. Otherwise, some kind of List would be much better. Therefore this could be a final field to underline the fact that it's fixed.